usmqe_tests/api/gluster/test_brick.py

```diff
@@ -0,0 +1,129 @@
+"""
+REST API test suite - gluster brick
+"""
+import pytest
+from usmqe.api.tendrlapi import glusterapi
+import usmqe.usmssh as usmssh
+
+
+LOGGER = pytest.get_logger('volume_test', module=True)
+"""@pylatest default
+Setup
+=====
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+
+def test_create_brick_valid(
+        valid_cluster_id,
+        valid_brick_name,
+        valid_devices,
+        valid_session_credentials):
+    """@pylatest api/gluster.create_brick_valid
+        API-gluster: create_brick
+        ******************************
+
+        .. test_metadata:: author fbalak@redhat.com
```

Comment:
It's from unmerged PR https://github.com/usmqe/usmqe-tests/pull/103/files#diff-0e248528164c56dc9d89b5c07001b38cR32